Python/ceval.c

```diff
@@ -5238,8 +5238,17 @@ import_from(PyThreadState *tstate, PyObject *v, PyObject *name)
         PyErr_SetImportError(errmsg, pkgname, NULL);
     }
     else {
+        _Py_IDENTIFIER(__spec__);
+        PyObject* spec = _PyObject_GetAttrId(v, &PyId___spec__);
```

Comment:
```suggestion
        PyObject *spec = _PyObject_GetAttrId(v, &PyId___spec__);
```

Reply:
this module is inconsistent at best as to whether it uses `PyObject *foo` / `PyObject* foo` / `PyObject * foo` -- does PEP7 have a preference?

Reply:
PEP7 doesn't have a direct preference but the code samples in it do put the asterisk with the variable name instead of the type. (https://www.python.org/dev/peps/pep-0007/#code-lay-out)

The rest of the code base tends to loosely follow that as well. `ceval` is one of those really old files so that's why it probably has a little inconsistency.